docusaurus/docs/reactnative/guides/message_input_customization.mdx

```diff
@@ -229,4 +232,164 @@ const additionalTextInputProps = useMemo(() => {
 >
   ...
 </Channel>
-```
\ No newline at end of file
+```
+
+## Replacing AttachmentPicker With Native Image Picker
+
+Attachment picker opens up as bottom sheet by default.
+It's easy to replace this default attachment picker with a device's native image picker, as shown in screenshots below.
+In this section, we will walk through the integration of native image picker to attach images in message.
+
+<table>
+  <tr>
+    <td align='center' width="33%"><img src={nativeAttachmentPickerStep1}/></td>
+    <td align='center' width="33%"><img src={nativeAttachmentPickerStep2}/></td>
+  </tr>
+  <tr></tr>
+  <tr>
+    <td align='center'>Attachment Picker actionsheet</td>
+    <td align='center'>Photo library</td>
+  </tr>
+</table>
+
+Channel component takes a prop [AttachButton](../core-components/channel.mdx#attachbutton),
+which can be used to override the default attach button next to input box as following. This prop can be used to
+override default `onPress` handler for the `AttachButton` component.
+
+```tsx
+import { AttachButton, Channel } from 'stream-chat-react-native';
+
+const CustomAttachButton = () => {
+  const onPressHandler = () => {
+    // Custom handling of onPress action on AttachButton
+  };
+
+  return <AttachButton handleOnPress={onPressHandler} />;
+}
+
+<Channel AttachButton={CustomAttachButton} />
+
+```
+
+Lets implement the `onPressHandler` to open action sheet showing options to choose attachment from Photo Library, Camera or Files.
```

Comment:
Maybe lets make it less personal? Instead of `Lets implement...` maybe something like `In order to make the onPressHandler open the action sheet showing the options to choose attachments from Photo Library, Camera or Files, you can use expo/react-native-action-sheet package for the action sheet implemented in this example`